packages/workspace/src/command-line/workspace-schematic.ts

```diff
@@ -219,37 +221,67 @@ async function executeSchematic(
   );
 
   let nothingDone = true;
+  let loggingQueue: string[] = [];
+  let error = false;
```

Comment:
My preference here would be for `hasError`, rather than simply `error` - the latter is what I'd normally assume to be the name of the error object itself.

This is a minor thing though.

Reply:
That makes a lot of sense. I'll make this change.

Reply:
Change implemented